src/test/java/uk/gov/companieshouse/api/accounts/transformer/StatementTransformerTest.java

```diff
@@ -0,0 +1,85 @@
+package uk.gov.companieshouse.api.accounts.transformer;
+
+
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.TestInstance.Lifecycle;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.companieshouse.api.accounts.model.entity.StatementDataEntity;
+import uk.gov.companieshouse.api.accounts.model.entity.StatementEntity;
+import uk.gov.companieshouse.api.accounts.model.rest.Statement;
+
+@ExtendWith(MockitoExtension.class)
+@TestInstance(Lifecycle.PER_CLASS)
+public class StatementTransformerTest {
+
+
+    private StatementTransformer statementTransformer = new StatementTransformer();
```

Comment:
As we are using mockito we could use `@InjectMocks` so that mockito instantiates it for us